commands/logout.go

```diff
@@ -94,30 +94,9 @@ func logout(args logoutArguments) error {
 		return nil
 	}
 
-	logoutResponse, apiResponse, err := Client.UserLogout(logoutActivityName)
-	if err != nil {
-
-		if apiResponse == nil || apiResponse.Response == nil {
-			return microerror.Mask(noResponseError)
-		}
-
-		if apiResponse.StatusCode == http.StatusForbidden {
-			return microerror.Mask(accessForbiddenError)
-		}
-
-		// special treatment for HTTP 401 (unauthorized) error,
-		// in which case no JSON body is returned.
-		if apiResponse.StatusCode == http.StatusUnauthorized {
-			return microerror.Mask(notAuthorizedError)
-		}
-
-		// other cases
-		return microerror.Maskf(unspecifiedAPIError, err.Error())
-	}
-
-	if logoutResponse.StatusCode != apischema.STATUS_CODE_RESOURCE_DELETED {
-		return microerror.Maskf(unspecifiedAPIError, "response: %v", logoutResponse)
-	}
+	ap := ClientV2.DefaultAuxiliaryParams()
+	ap.ActivityName = logoutActivityName
 
-	return nil
+	_, err := ClientV2.DeleteAuthToken(args.token, ap)
```

Comment:
It would be good to mask the error here in case there is any, it makes debugging easier by providing consistent stack trace where we can look up the code function by function. If we skip masking the error here, this function will not show in stack trace and there would be a missing step which feels wrong.

Reply:
Done